src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -675,6 +680,19 @@ private void processLogsAndUpload() throws InterruptedException {
         }
     }
 
+    /**
+     * This method is used for tests monitoring the event status in the service.
```

Comment:
Don't mention it is used for tests. Yes, right now it is used for tests but it could be used internally as well for things like sending metrics when something happens etc etc.